sdk/redisenterprise/Azure.ResourceManager.RedisEnterprise/src/autorest.md

```diff
@@ -126,5 +128,7 @@ directive:
       $.OperationStatus.properties.error['x-ms-client-name'] = 'ErrorResponse';
       $.OperationStatus.properties.startTime['format'] = 'date-time';
       $.OperationStatus.properties.endTime['format'] = 'date-time';
+      $.ImportClusterParameters.properties.sasUris['format'] = undefined;
+      $.MaintenanceWindow.properties.duration['format'] = 'duration';
```

Comment:
why we have to remove them?
do we have the same format on previous version's swagger?
I think for `sasUris` because of the `format-by-name-rules` configuration, the generated code of this should be `IList<Uri>`

Reply:
Yes, The current SDK is generated as `IList<Uri>`. The migration spec PR defined this format as `password` , which is not supported by the modeler. Looks like it comes from `@secret` in TypeSpec and cannot be changed. So remove this format and leave it handld by our code gen.

Reply:
Added comments for that